features/project/source/browse_files.feature

```diff
@@ -133,6 +143,16 @@ Feature: Project Source Browse Files
     Then I am redirected to the files URL
     And I don't see the ".gitignore"
 
+  @javascript
+  Scenario: I can replace file and commit
+    Given I click on ".gitignore" file in repo
+    And I see the ".gitignore"
+    And I click on "Replace"
+    When I replace it with "LICENSE"
+    And I fill the commit message
+    And I click on "Replace file"
+    Then I am redirected to the ".gitignore"
+
```

Comment:
@tsigo @stanhu thanks, as you can see I already use attach_file in the browse_file.rb, however, after attach file to the :file_upload, and then when after click button "Upload file", it is not redirect to the blob path of the uploaded file, I have not idea why it happens here. Except file the commit message, do  I need to do some other actions after attach file and before click the "Upload file" action?